src/storm/solver/LpMinMaxLinearEquationSolver.cpp

```diff
@@ -65,9 +65,11 @@ namespace storm {
             // Add a constraint for each row
             for (uint64_t rowGroup = 0; rowGroup < this->A->getRowGroupCount(); ++rowGroup) {
                 // The rowgroup refers to the state number
-                if (this->choiceFixedForRowGroup && this->choiceFixedForRowGroup.get()[rowGroup]) {
-                    // The choice is fixed
-                    auto rowIndex = this->A->getRowGroupIndices()[rowGroup] + this->getInitialScheduler()[rowGroup];
+                for (uint64_t rowIndex = this->A->getRowGroupIndices()[rowGroup]; rowIndex < this->A->getRowGroupIndices()[rowGroup + 1]; ++rowIndex) {
+                    if (this->choiceFixedForRowGroup && this->choiceFixedForRowGroup.get()[rowGroup]) {
+                        // The choice is fixed so we set the rowIndex to the fixed one
+                        auto rowIndex = this->A->getRowGroupIndices()[rowGroup] + this->getInitialScheduler()[rowGroup];
+                    }
```

Comment:
The `auto` in line 71 creates a separate variable `rowIndex` that runs out of scope in line 72. I guess you wanted to set the `rowIndex` variable from line 68 instead? Anyway, I'd suggest to do it like this:
```suggestion
                uint64_t rowIndex, rowGroupEnd;
                if (this->choiceFixedForRowGroup && this->choiceFixedForRowGroup.get()[rowGroup]) {
                    rowIndex = this->A->getRowGroupIndices()[rowGroup] + this->getInitialScheduler()[rowGroup];
                    rowGroupEnd = rowIndex + 1;
                } else {
                    rowIndex = this->A->getRowGroupIndices()[rowGroup];
                    rowGroupEnd = this->A->getRowGroupIndices()[rowGroup + 1];
                }
                for (; rowIndex < rowGroupEnd; ++rowIndex) {
```
Then, the `if` block in line 87 below can be removed and we check for fixed choices less often.